### PR TITLE
include OBS install steps for CentOS

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@TomSweeneyRedHat @rhatdan PTAL

### CODE REVIEW

File: install.md [hunk 1]
```diff
@@ -7,12 +7,45 @@ RHEL/CentOS ≥ 8 or Fedora you can install it using:
 $ sudo dnf install skopeo
 ```
 
+Newer skopeo releases may be available on the repositories provided by the
+Kubic project. Beware, these have not been blessed by the official Red Hat QA
```
Comment (rhatdan) [inline install.md:11]: Since this is not Red Hat, then we should not mention them. Nothing in Centos goes through Red Hat QA.

**Comment by TomSweeneyRedHat** [comment]
LGTM other than @rhatdan 's comment

File: install.md (hunk 1 above)
Comment (lsm5) [reply install.md:11]: ah yes, i'll check what I have in podman and update this one.

Comment (lsm5) [reply install.md:11]: so the podman.io repo mentions Red Hat QA as well. So it needs to be changed there as well in that case.

**Comment by lsm5** [comment]
PTAL @ update.

File: install.md [hunk 2]
```diff
@@ -7,12 +7,43 @@ RHEL/CentOS ≥ 8 or Fedora you can install it using:
...
 ```sh
 $ sudo yum install skopeo
 ```
 
+Newer skopeo releases may be available on the repositories provided by the
+Kubic project. Beware, these may not be suitable for production environments.
+
+```sh
+$ sudo curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/CentOS_7/devel:kubic:libcontainers:stable.repo
+$ sudo yum -y install podman
```
Comment (mtrmac) [inline install.md:44]: `install `**`podman`**?

**Review by mtrmac — COMMENTED** [review]
The page is getting a bit long and difficult to skim. Could you add
```markdown
## Distribution Packages
```
and 
```markdown
## Building from Source
```
headers, and `###`-level headers for the different distributions, please?

File: install.md [hunk 3]
```diff
@@ -7,12 +7,43 @@ RHEL/CentOS ≥ 8 or Fedora you can install it using:
 $ sudo dnf install skopeo
 ```
 
+Newer skopeo releases may be available on the repositories provided by the
```
Comment (TomSweeneyRedHat) [inline install.md:10]:
```suggestion
Newer Skopeo releases may be available on the repositories provided by the
```

File: install.md [hunk 4]
```diff
@@ -7,12 +7,43 @@ RHEL/CentOS ≥ 8 or Fedora you can install it using:
...
+sudo dnf -y install skopeo
+```
+
 on RHEL/CentOS ≤ 7.x:
 
 ```sh
 $ sudo yum install skopeo
 ```
 
+Newer skopeo releases may be available on the repositories provided by the
```
Comment (TomSweeneyRedHat) [inline install.md:39]:
```suggestion
Newer Skopeo releases may be available on the repositories provided by the
```

File: install.md (hunk 2 above)
Comment (lsm5) [reply install.md:44]: ha, copy-pasta, i'll fix it.

**Comment by lsm5** [comment]
PTAL at updates. I made some more changes (for example curl instead of wget) as that was requested by someone on the podman docs.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**